src/find_meaningful_words.py

```diff
@@ -0,0 +1,143 @@
+from collections import Counter
+from src.utils import SaveRead
+from src.count_words import get_words_only_small_letters, count_words_from_file
+
+#
+WORDS_PLACE_IN_FREQUENCY_LIST = 0
+FREQUENCY_PLACE_IN_FREQUENCY_LIST = 1
+
+# TODO: Add documentation to each function.
+# TODO: Add tests to each function
+# TODO: Divide function into different files.
+# TODO: Add configuration file for magic numbers and strings.
+
+
+def sort_words_count(counted_words_tuple_list):
+    sorted_counted_words_tuple_list = sorted(
+        counted_words_tuple_list, key=lambda x: x[FREQUENCY_PLACE_IN_FREQUENCY_LIST], reverse=True)
+    return sorted_counted_words_tuple_list
+
+
+def count_words_in_list(words_list):
+    counted_words_dict = dict(Counter(words_list))
+    counted_words_tuple_list = counted_words_dict.items()
+    sorted_counted_words_tuple_list = sort_words_count(
+        counted_words_tuple_list)
+    return sorted_counted_words_tuple_list
+
+
+def get_only_common_words(words_frequencies_dict, number_of_most_common):
```

Comment:
Thanks!! Changed both
I think I can be even more specific and maybe that can be discussed. 
Because I'm dealing with words from the context I wrote words in the function. 
But actually it words for any elements and counts tuple list 
What do you think? Would you change it ? to your experience